spec/fixtures/deployments.json

```diff
@@ -0,0 +1,116 @@
+[
```

Comment:
This file was pulled from the https://docs.gitlab.com/ce/api/deployments.html#get-a-specific-deployment documentation